providers/base/units/mtd/jobs.pxu

```diff
@@ -2,17 +2,24 @@ id: mtd/check-total-count
 category_id: mtd
 _summary: Check the total MTD count for the platform
 _description:
-  Check the total MTD count for the platform, relies on the user specifying the number of mtd. 
+  Check the total MTD count for the platform, relies on the user specifying the count of mtd.
   Usage of environment variable: TOTAL_MTD_COUNT={count}
   e.g. TOTAL_MTD_COUNT=2
 plugin: shell
 user: root
 imports: from com.canonical.plainbox import manifest
-requires: manifest.has_mtd == 'True'
+requires:
+  manifest.has_mtd == 'True'
+  package.name == "mtd-utils"
 flags: also-after-suspend
 estimated_duration: 5
 environ: TOTAL_MTD_COUNT
-command: mtd.sh count "$TOTAL_MTD_COUNT"
+command:
+  if [ -z "$TOTAL_MTD_COUNT" ]; then
+    echo "Missing the 'TOTAL_MTD_COUNT' environment variable. Please define it in checkbox.conf"
```

Comment:
It can also be defined in a launcher, so maybe something more generic like

> Please define it in the [environment] section of your Checkbox configuration.

would be better?